.github/workflows/s3-tests.yml

```diff
@@ -84,90 +106,50 @@ jobs:
       - run: go version
 
       - name: Set up Python
-        uses: actions/setup-python@v5
+        uses: actions/setup-python@v4
```

Comment:
Is v4 more suitable?

Reply:
new is always better :) it was a copypaste from somewhere